components/UI/InviteUsersToMatrixRoom/index.js

```diff
@@ -102,10 +103,10 @@ export const InviteUserToMatrixRoom = ({ roomId, onSuccess }) => {
 
         // if everything is okay, we let the user know and exit the view.
         successAmount > 0 && setUserFeedback(<Trans t={t} i18nKey="invitedUser" count={successAmount}>{ { successAmount } } user was invited and needs to accept your invitation</Trans>);
-        await new Promise(() => setTimeout(() => {
+        await waitFor(() => {
```

Comment:
**Second example**

This here would just be:
```js
_.delay(() => {
    clearInputs();
    if (onSuccess && successAmount === selectedUsers.length) onSuccess();
}, 3000);
```

Reply:
... and so on.
I think the same thing works for all instances of `waitFor()` below. Correct me if I am wrong though.

Reply:
yeah looks like it does exactly the same, i'll change them